tests/Fixtures/Integration/priority/multiline_string_to_heredoc,escape_implicit_backslashes.test

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Integration of fixers: multiline_string_to_heredoc,escape_implicit_backslashes.
```

Comment:
how can I fix:
```
1) PhpCsFixer\Tests\AutoReview\FixerFactoryTest::testPriorityIntegrationTestFilesAreListedInPriorityGraph with data set #290
Missing priority test entry for file "multiline_string_to_heredoc,escape_implicit_backslashes.test".
Failed asserting that false is true.

/home/runner/work/PHP-CS-Fixer/PHP-CS-Fixer/tests/AutoReview/FixerFactoryTest.php:166

2) PhpCsFixer\Tests\AutoReview\FixerFactoryTest::testFixerWithNoneDefaultPriorityIsTested
Fixers with non-default priority and yet without priority unit tests [vide "getFixersPriorityGraph()" and "getPhpDocFixersPriorityGraph()"]: "multiline_string_to_heredoc."
Failed asserting that an array is empty.

/home/runner/work/PHP-CS-Fixer/PHP-CS-Fixer/tests/AutoReview/FixerFactoryTest.php:326
```

Reply:
1) Add entry to `FixerFactoryTest::getFixersPriorityGraph()`
2) I am not sure, but maybe it's related to first issue and will disappear after updating graph

Reply:
Thank you!